client/components/chart/index.jsx

```diff
@@ -60,29 +61,27 @@ module.exports = React.createClass( {
 	},
 
 	resize: function() {
-		if ( this.isMounted() ) {
```

Comment:
if we still allow calling when not mounted, we should be able to remove the listener in `componentWillUnmount` - @Copons is this already handled for us?